Mathlib/Analysis/Calculus/FDeriv/Symmetric.lean

```diff
@@ -32,6 +33,11 @@ requiring that the point under consideration is accumulated by points in the int
 These are written using ad hoc predicates `IsSymmSndFDerivAt` and `IsSymmSndFDerivWithinAt`, which
 increase readability of statements in differential geometry where they show up a lot.
 
+We also deduce statements over an arbitrary field, requiring that the function is `C^2` if the field
+is `ℝ` or `ℂ`, and analytic otherwise. Formally, we assume that the function is `C^n`
+with `minSmoothness 𝕜 2 ≤ n`, where `minSmoothness 𝕜 i` is `i` if `𝕜` is `ℝ` or `ℂ`,
+and `ω` otherwise.
```

Comment:
Apparently [This answer](https://math.stackexchange.com/a/69231/12932) says that a C^1 function on a Banach space over ℂ is already analytic, but it seems Mathlib only knows the 1-dimensional case. And it seems you don't need completeness in this PR ...

Reply:
The point of the PR is a little bit different: We already have the statement over R or C (assuming C^2), and I want to extend it to other fields (assuming analyticity, because otherwise it's not true). You are correct that one could do better over C (where C^1 implies analytic) but I don't think it would really be useful to state it in this way (because, after all, the C^1 functions over C are already analytic, so the current lemma applies).